src/main/scala/akka/persistence/cassandra/query/javadsl/CassandraReadJournal.scala

```diff
@@ -145,5 +147,45 @@ class CassandraReadJournal(scaladslReadJournal: akka.persistence.cassandra.query
    */
   def currentEventsByTag(tag: String, offset: UUID): Source[UUIDEventEnvelope, Unit] =
     scaladslReadJournal.currentEventsByTag(tag, offset).asJava
+
+  /**
+    * `eventsByPersistenceId` is used to retrieve a stream of events for a particular persistenceId.
+    *
+    * In addition to the `offset` the `EventEnvelope` also provides `persistenceId` and `sequenceNr`
```

Comment:
I will do that, it makes sense from user's perspective. I assumed the offset should always start from 0 for any query as a 'standard'.